src/public-lib/components/checkedSelect/CheckedSelect.tsx

```diff
@@ -19,6 +19,7 @@ type CheckedSelectProps = {
     value: {value: string}[];
     options: Option[];
     placeholder?: string | JSX.Element;
+    selectComponents?: {[componentType: string]: (props: any) => JSX.Element};
```

Comment:
i think selectComponents is a confusing name, reactSelectComponents would be better IMO